entries/r/remotedesktopmanager.com.json

```
@@ -5,7 +5,7 @@
     "tfa": [
       "totp"
     ],
```

Looks like multiple 2FA-methods are available:
```suggestion
      "totp",
      "custom-software",
      "custom-hardware"
    ],
    "custom-software": [
      "Duo"
    ],
    "custom-hardware": [
      "Yubico OTP"
    ],
```